manufacturing/internals/README.md

```diff
@@ -133,7 +145,7 @@ You can now connect all the wiring (except main power):
 * connect the pinch valves to their driver pigtails as follows:
   * top - inhale
   * bottom - exhale
-* Secure sensor tubing and wiring with zip ties `[A25]`
+* Secure sensor tubing and wiring with zip ties `[A24]`
```

Comment:
Suggestion: Don't tighten the zip ties to point where the fluid flow is restricted in the tubes or where the wires can't move at all.